appshell/client_handler_mac.mm

```diff
@@ -56,9 +56,12 @@
   NSWindow* window = [view window];
   std::string titleStr(title);
   NSString* str = [NSString stringWithUTF8String:titleStr.c_str()];
-  [window setTitle:str];
```

Comment:
@nethip 
Title disappeared while working with two windows (via Debug > New Brackets Window).
Both windows had two separate projects open. Getting Started folder in one window and some other folder in the other window. Entered full screen mode in both windows. This is what happened thereafter.
![screen shot 2016-05-12 at 11 19 29 pm](https://cloud.githubusercontent.com/assets/2939847/15224618/16f0142e-1898-11e6-97ab-0521ccd3d655.png)